React-Homeworks/src/components/FoodCard/FoodCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import "./FoodCard.css";
+import Button from "../Buttons/Buttons";
+
+function FoodCard({ title, subTitle, price, imgUrl }) {
+  const [quantity, setQuantity] = useState("1"); 
+
+  const handleChange = (event) => {
+    const newQuantity = event.target.value;
+    setQuantity(newQuantity);
+  };
+
+  return (
+    <figure className="card flex-elem">
+      <div className="card__wrapper flex-elem">
+        <div className="card__div-img flex-elem">
+          <img className="card__img" src={imgUrl} alt="Pam" />
+        </div>
+        <figcaption className="flex-elem">
+          <h2 className="card__h2 flex-elem">
```

Comment:
You need to be really careful with heading tag usage. They should fallow some rules like nesting, one by one usage and etc.